examples/petclinic/utils.bal

```diff
@@ -0,0 +1,92 @@
+import ballerina/http;
+import ballerina/time;
+
+public isolated function createNotFoundProblem(string detail, string problemType = "/problems/not-found", string? instance = ()) returns NotFoundProblem {
+    return {
+        body: {
+            'type: problemType,
+            title: "Not Found",
+            status: http:STATUS_NOT_FOUND,
+            detail,
+            instance,
+            timestamp: time:utcNow()
+        }
+    };
+}
+
+public isolated function createBadRequestProblem(string detail, json? validationErrors = (), string? instance = ()) returns BadRequestProblem {
+    ProblemDetails problemDetails = {
+        'type: VALIDATION_ERROR_TYPE,
+        title: "Bad Request",
+        status: http:STATUS_BAD_REQUEST,
+        detail,
+        instance,
+        timestamp: time:utcNow()
+    };
+
+    if validationErrors is json {
```

Comment:
This will be always true, Ballerina `json` type includes `null` type as well. So the `json? validationErrors` is same as defining `json validationErrors`. I think the functionality here is not check that this is not null right?

[1] https://ballerina.io/learn/by-example/json-type/

```suggestion
    if validationErrors !is () {
```